### PR TITLE
fix: Hide Event Map without coordinates

### LINKED ISSUE #2565: UI and Maps Integration Bug
In the app, UI needs improvement. And, when we click on a particular event, the integrated map is empty, after tapping on it, Google maps gets opened but the location of the event is not showed.

### PR DESCRIPTION
**Pull request by scixor**
Fixes #2565 

## Changes: 
- There was an error where the coordinates were being set to 0.000 as a double which bypassed the current check of `lattitude != null` etc.
- Now the UI only shows the Picasso picture when there is some coordinate that eliminates the possibility of clicking it to go to Google maps

## Screenshots for the change:
When location is present: 

image

When Location isn't present:
image

The Zip containing a test video and the debug.apk
[MapBug.zip](https://github.com/fossasia/open-event-attendee-android/files/4085772/MapBug.zip)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-android-general/pullRequest?prid=4850028&bid=16094375)

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 1]
```diff
@@ -457,7 +457,7 @@ class EventDetailsFragment : Fragment() {
         // load location to map
         val mapClickListener = View.OnClickListener { startMap(event) }
 
-        if (!event.locationName.isNullOrEmpty() && event.longitude != null && event.latitude != null) {
+        if (!event.locationName.isNullOrEmpty() && event.longitude != null && event.longitude != 0.00 && event.latitude != null && event.latitude != 0.00) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: Codacy Issue found: [ComplexCondition - 5/4 - [loadEvent] at /src/app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt:460:13](https://app.codacy.com/manual/fossasia/open-event-android-general/pullRequest?prid=4850028)

Comment (Sid911) [reply app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: sure I thought it was too messy and will be pointed out... I will fix it 😇

File: app/src/main/res/layout/content_event.xml [hunk 2]
```diff
@@ -361,7 +361,7 @@
                     android:id="@+id/imageMap"
                     android:layout_width="match_parent"
                     android:layout_height="@dimen/home_logo_width"
-                    app:hideIfEmpty="@{event.locationName}"
+                    android:visibility="visible"
```
Comment (liveHarshit) [inline app/src/main/res/layout/content_event.xml]: Why? It will be visible by default.

Comment (scixor) [reply app/src/main/res/layout/content_event.xml]: actually it isn't when I checked the application ... 😇, Also there is already a check for locationName when the view is being created so it isn't really required we are just adding multiple checks for no reason...

**Comment by scixor** [comment]
aaah build failed? I guess formatting 😅

File: app/src/main/res/layout/content_event.xml (hunk 2 above)
Comment (scixor) [reply app/src/main/res/layout/content_event.xml]: I have removed that statement too as, it is set visible by default 😅

**Comment by scixor** [comment]
@liveHarshit @iamareebjamal Done 😁

**Review by liveHarshit — DISMISSED** [review]

**Comment by scixor** [comment]
The formatting............. 😌 means I have to reformat the entire file......... sorry to ping again @liveHarshit  I might have to stale the review

**Review by liveHarshit — CHANGES_REQUESTED** [review]
Revert the unrelated changes

**Comment by scixor** [comment]
sure but even after entire reformatting of the page... the build isn't sucessful because of linting?

**Comment by scixor** [comment]
I really hope this makes through the linting 😅

**Comment by Sid911** [comment]
I really hope this makes through the linting 😅

**Comment by scixor** [comment]
@liveHarshit @iamareebjamal everything is done.. and fortunately the linting forgave me this time around 😂

**Review by liveHarshit — APPROVED** [review]

**Comment by scixor** [comment]
@liveHarshit can this merged 😅.. I want to get this done as soon as possible

**Pull request merged**